config.toml

```diff
@@ -102,11 +123,29 @@ description = "Production-Grade Container Orchestration"
 languageName ="English"
 # Weight used for sorting.
 weight = 1
-[languages.cn]
+
+[languages.zh]
 title = "Kubernetes"
 description = "Production-Grade Container Orchestration"
```

Comment:
Should this description use Chinese? Same concern for below languages.

Reply:
If the team would like to submit a PR for Chinese strings after this PR merges, that would be awesome. 👍